tests/cmd/syntax_highlighting_css.in/_cobalt.yml

```diff
@@ -0,0 +1,2 @@
+syntax_highlight:
```

Comment:
Could you add this test case in a previous commit but use a regular theme for it?  Then this commit will switch to the `css` theme and we get to see how the behavior changes

Reply:
https://github.com/cobalt-org/cobalt.rs/commit/36fe05270d3063fe3f900ca0a49b33d19f1929e2#diff-3b6b0a70768bbae583638199ee4a7ff1cab39d68eeb4a75d9339fd2b86956673